Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -495,6 +464,21 @@ static void OnConnected(int connectionId)
                 // add connection
                 NetworkConnectionToClient conn = new NetworkConnectionToClient(connectionId);
                 OnConnected(conn);
+
+                // was this the connection to the local host?
+                if (pendingLocalConnection)
```

Comment:
Is this really simpler?  keeping this extra state around?

Reply:
How do you know this connection is the local one? this connection could be coming from anywhere. Are you just hoping that the local connection wins the race?